### ISSUE TITLE
Add Splunk_hec label to metrics output

### ISSUE THREAD

**Issue by FluffySocks32** [redpanda-data/connect#2609]
Hi, 

When using the splunk_hec output the label fails to be added, resulting in the field showing up as null from the statsd output. 

I added the following below [line 107](https://github.com/benthosdev/benthos/blob/main/internal/impl/splunk/template_output.yaml#L107) of the splunk_hec template, and it successfully adds the label back in. 

`
metrics_mapping: |
  #!blobl
  meta label = $label | ""
`

Long story short, I can't add this in as a PR. Would it be possible to confirm this is actually a fix? 

Thank you

**Comment by mihaitodor** [comment]
Hey @FluffySocks32 👋 

> Long story short, I can't add this in as a PR. 

Why not? Feel free to create a fork and send a PR from there.

> Would it be possible to confirm this is actually a fix?

Not sure how. Someone would have to test it with a Splunk account, since there are no integration tests and not sure if you can run a standalone Splunk emulator as a Docker container. Also, in the Splunk {`template_output.yaml`](https://github.com/benthosdev/benthos/blob/43517e6ed43167ccb29477fe1f6666d742be5f94/internal/impl/splunk/template_output.yaml) file you linked has no statement containing `label`, so using `$label` in that `metrics_mapping` looks wrong, but I guess I'm missing something...

**Comment by twmb** [comment]
Closing as stale — more information was requested but no follow-up was provided. If you can describe the specific labeling behavior needed, feel free to open a new issue or PR.
